docs/index.rst

```diff
@@ -6,7 +6,9 @@ Hello, welcome to Contessa documentation!
 Quick Example
```

Comment:
I miss some general description of the project, for user to understand the purpose at first glance. There is some info in `quick_start`, but not very much.

Reply:
Yep, thats true, gonna add some more to readme and link it in index :)